zer0-os/zOS
# feat(conversation-header): implement last seen at text to subtitle

### What does this do?
- adds last seen at text to subtitle

### Why are we making this change?
- as per design

### How do I test this?
- run tests as usual
- run ui and check conversation header subtitle text

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?


<img width="609" alt="Screenshot 2024-04-19 at 00 06 55" src="https://github.com/zer0-os/zOS/assets/39112648/4a641a53-ea0d-4ad3-926e-7ba0f5cebced">
